packages/edit-site/src/components/block-editor/index.js

```diff
@@ -49,6 +50,7 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 			useSubRegistry={ false }
 		>
 			<BlockEditorKeyboardShortcuts />
+			<TemplatePartConverter />
```

Comment:
This component and its corresponding file seem unnecessary if we are fine converting from the standard block transform flow:
![Screen Shot 2020-10-20 at 5 08 45 PM](https://user-images.githubusercontent.com/28742426/96644567-4217a500-12f7-11eb-89b7-80a6c1350235.png)

Reply:
We will need this as we got rid of the transform flow since this should only be available through the ellipsis menu - https://github.com/WordPress/gutenberg/pull/20445#issuecomment-713499116